### ISSUE TITLE
Users Endpoint Updates

### ISSUE THREAD

**Issue by HankG** [diaspora/api-documentation#31]
- [x] ~~Name field should be broken into first_name and last_name since name is a synthetic field and setting/getting symmetry is preferred.~~
- [x] NSFW toggle on profile should be added.
- [ ] Need to add error return paths to the documentation.

**Comment by SuperTux88** [comment]
> Name field should be broken into first_name and last_name since name is a synthetic field and setting/getting symmetry is preferred.

No, because they will be combined in the future in diaspora, see https://github.com/diaspora/diaspora/issues/3648.

**Comment by HankG** [comment]
> > Name field should be broken into first_name and last_name since name is a synthetic field and setting/getting symmetry is preferred.
> 
> No, because they will be combined in the future in diaspora, see [diaspora/diaspora#3648](https://github.com/diaspora/diaspora/issues/3648).

If we go that route, which is fine even though I've already implemented it this way, it will be inconsistent with the current UI.  When is that changing and do we not care about the inconsistency in the interim?

**Comment by jhass** [comment]
* [ ] Block / Ignore user

(Or am I blind?)
